tests/cross/src/test/scala/tests/pc/CompletionDocSuite.scala

```diff
@@ -537,25 +536,7 @@ class CompletionDocSuite extends BaseCompletionSuite {
        |- `fin`: Finally logic which if defined will be invoked after catch logic
        |Catch - scala.util.control.Exception
        |""".stripMargin,
-    includeDocs = true,
-    compat = Map(
-      "2.13" ->
-        """|> A container class for catch/finally logic.
-           |
-           | Pass a different value for rethrow if you want to probably
-           | unwisely allow catching control exceptions and other throwables
-           | which the rest of the world may expect to get through.
-           |
-           |**Type Parameters**
-           |- `T`: result type of bodies used in try and catch blocks
-           |
-           |**Parameters**
-           |- `fin`: Finally logic which if defined will be invoked after catch logic
```

Comment:
In this and above the difference was actually only the parameter order, which is different when run with Scala 2.13 and 2.12. Probably due to reimplementation of standard collections. The tests now show what will really be in Metals, which is run in 2.12.